dpdata.py

```diff
@@ -159,6 +160,12 @@ def __fixup_nomail_flag(self):
                 # Empty out NOMAIL_REASON if NOMAIL not set
                 dp_donorrecord['NOMAIL_REASON'] = ''
 
+    def __copy_spouse_email(self):
```

Comment:
@kdawgsf, did you get any special notification after my comment above? I'm trying out these new review tools introduced by GitHub that should produce a better workflow.